### PR TITLE
PD-1501 date_added field for solr, and mgmt command for checking tasks

### PR DESCRIPTION
**Pull request by mklauber**
start indexing the date_added field (already part of the production schema). 
Management command to check if tasks are backing up, but not making it to Solr.  (Dependent on the date_added field).

### CODE REVIEW

File: import_jobs.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def get_jobsfs_zipfile(guid):
     authheader = "Basic %s" % base64.encodestring('%s:%s' % ('microsites',
                                                              'di?dsDe4'))
     req.add_header("Authorization", authheader)
-    resp = urllib2.urlopen(req)
+    resp = urllib2.urlopen(req, timeout=30)
```
Comment (aspidites) [inline import_jobs.py]: Do we already/need to do something if a timeout occurs?

File: seo/management/commands/check_tasks.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    solr_cuttoff = 0
+    look_behind = timedelta(hours=1)
+    
+
+    def handle(self, port="15672", queues="solr,priority", *args, **options):
+        # Determine the number of tasks still in Rabbit 
+        msg_count = 0
+        for queue in queues.split(','):
+            # Get the queue data from the rabbit Management API
+            uri = "http://%(broker)s:%(port)s/api/queues/dseo-vhost/%(queue)s" % {'broker': settings.BROKER_HOST,
```
Comment (aspidites) [inline seo/management/commands/check_tasks.py]: nit: alignment

File: seo/management/commands/check_tasks.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                                                                                'port': port,
+                                                                                'queue': queue}
+            resp = requests.get(uri, auth=(settings.BROKER_USER, settings.BROKER_PASSWORD))
+            data = json.loads(resp.content)
+            msg_count += data["messages_ready"]
```
Comment (aspidites) [inline seo/management/commands/check_tasks.py]: Is this guaranteed to be an integer, or can it be something like "32"?

**Comment by aspidites** [comment]
Looks mostly fine. The "nit" insn't really important.

File: import_jobs.py (hunk 1 above)
Comment (mklauber) [inline import_jobs.py]: No.  We're currently using the default timeout, but we're getting strange hanging behavior.  If a timeout occurs, it should raise an exception, which should result in the task being cancelled and retried in 3 minutes.  Hopefully, by then whatever is causing connection issues should be resolved.

File: seo/management/commands/check_tasks.py (hunk 3 above)
Comment (mklauber) [inline seo/management/commands/check_tasks.py]: Api returns an int in the json.  That should be stable.

**Comment by mklauber** [comment]
Still want @JLMcLaugh to look it over to confirm it works for him.

File: import_jobs.py (hunk 1 above)
Comment (aspidites) [inline import_jobs.py]: :+1:

**Comment by aspidites** [comment]
I'm good with this. @galitskyd, @tdphillips , @JLMcLaugh ?

**Pull request merged**